cni/cmd/istio-cni/main.go

```diff
@@ -118,7 +118,26 @@ func parseConfig(stdin []byte) (*PluginConf, error) {
 }
 
 // cmdAdd is called for ADD requests
-func cmdAdd(args *skel.CmdArgs) error {
+func cmdAdd(args *skel.CmdArgs) (err error) {
+	// Defer a panic recover, so that in case if panic we can still return
+	// a proper error to the runtime.
+	defer func() {
+		if e := recover(); e != nil {
+			msg := fmt.Sprintf("istio-cni panicked during cmdAdd: %v", e)
+			log.Errorf(msg)
```

Comment:
I don't think we should log it here and at line 132. They were all duplicated. The recover block should only construct the crash message and the log should only happen at line 137.

Reply:
Thanks